include/insns/RV32I.h

```diff
@@ -911,63 +981,63 @@ namespace SST{
       //            <rs2Class> <rs3Class> <format> <func> <nullEntry>
       // ----------------------------------------------------------------------
       std::vector<RevInstEntry> RV32ITable = {
-      {RevInstEntryBuilder<RevInstDefaults>().SetMnemonic("lui %rd, $imm"  ).SetCost(1).SetOpcode(0b0110111).SetFunct3(0b0).SetFunct7(0b0).SetrdClass(RegGPR).Setrs1Class(RegUNKNOWN).Setrs2Class(RegUNKNOWN).Setrs3Class(RegUNKNOWN).Setimm12(0b0).Setimm(FUnk).SetFormat(RVTypeU).SetImplFunc(&lui ).InstEntry},
```

Comment:
why are you rewriting the entire instruction table.  this is unnecessary

Reply:
I reformated the instruction table for easy debugging. I noticed that in the previous instruction table, it is hard to check functX fields and immediate fields with the RISC-V specifications.